### ISSUE TITLE
Book metadata field missing ISBN

### ISSUE THREAD

**Issue by icf20** [MediaBrowser/Emby#1882]
hello, in my opinion there should be a way to edit and add the ISBN to a book

**Comment by LukePulverenti** [comment]
I'm closing this right now. Adding the field to the editor would be misleading because users would end up thinking the value would actually be used for something. We're not going to add the value until something actually uses it.

If you or anyone would like to contribute by adding a metadata provider to our books plugin, I think that would be fantastic. If one of those providers can make use of ISBN, then we'll be happy to add it to the editor.

Thanks !

**Comment by icf20** [comment]
@LukePulverenti do you want to open this now that version 4 have the audiobooks category?

edit: if anyone want to see how ISBN lookup can be done go to this project http://tellico-project.org/data-sources  also note tellico have the option to let their affiliate ID to the amazonlink to get some commissions :heavy_dollar_sign:

**Comment by LukePulverenti** [comment]
Not until we have a use for the data.

**Comment by icf20** [comment]
@LukePulverenti i will open a new issue specific for the audiobooks to make sure we dont get confused

**Comment by LukePulverenti** [comment]
There is now an OpenLibrary plugin in the Emby Plugin catalog for books, and it supports ISBN lookup.

Please try it out and report your experience. Thanks !
